tiledb/sm/config/config.h

```diff
@@ -157,6 +157,9 @@ class Config {
   /** Whether or not to use the refactored readers. */
   static const std::string SM_USE_REFACTORED_READERS;
 
+  /** Maximum memory budget for the sparse global order reader. */
```

Comment:
This memory budget eventually will be for all readers and writers. Not sure if we will remember to edit in subsequent PRs.